### DISCUSSION TITLE
Is DuckDB V1.0 coming soon? Then what're the important features?

### DISCUSSION THREAD

**Discussion by huxiaoxiang** [duckdb/duckdb#4239] [Show and tell]
I saw a in "DuckDB Testing - Present and Future"
[https://www.youtube.com/watch?v=BgC79Zt2fPs](url)
![image](https://user-images.githubusercontent.com/4969695/181669349-5d4b4c7e-7667-40fe-8f86-8eedf1549cb0.png)
Just curious what features will we get😄 @Mytherin

**Comment by hannes** [comment]
There are no crazy features planned for 1.0 but we do want to stabilise the on-disk data format before.

**Reply by huxiaoxiang** [reply]
> 

Got it, thanks hannes.
